src/compiler/transformers/module/module.ts

```diff
@@ -357,6 +357,9 @@ namespace ts {
                 return undefined;
             }
 
+            // Set emitFlags on the name of the classDeclaration
+            // This is so that when printer will not substitute the identifier

```

Comment:
1. Should it be "Set emitFlags on the name of the importEqualsDeclaration" ?
2. I think the second line should be reworded as "This is so the printer ..." ('that when' -> 'the')